doc/doxygen/src/getting-started.md

```diff
@@ -239,6 +239,16 @@ serial interface:
 make term BOARD=samr21-xpro PORT=/dev/ttyACM1
 ~~~~~~~~
 
+For flashing and accessing the board via the serial interface, the current user
+needs to have the correct access rights on the serial device.
+The easiest way to ensure this is to add the current user to the group that is
+owning the serial device. For example, this can be achieved on Linux by issuing
+the following line, logging out and logging in again:
+
+~~~~~~~~ {.sh}
+sudo usermod -aG $(stat --format="%G" /dev/ttyACM0) $USER
```

Comment:
Is that really persistent? It's also very brittle if there are multiple serial devices.

I don't know about other distributions, but on Debian/Ubuntu users in the `dialout` group can access serial devices. So a better solution would be

```suggestion
    sudo adduser $USER dialout
```

Reply:
> Is that really persistent? It's also very brittle if there are multiple serial devices.

It is persistent and works as long as all serial devices are owned by the same usergroup.

> I don't know about other distributions, but on Debian/Ubuntu users in the `dialout` group can access serial devices.

In the student laptop (Arch iirc) it was `uucp`. That's why the suggested line uses `stat --format="%G" /dev/ttyACM0` to find out the owner group.

`usermod -aG` and `adduser` both add a user to a group, not sure which one to prefer (preferably the one that is more likely to be installed per default on many systems).